electricity/src/addresses.rs

```diff
@@ -1,11 +1,14 @@
 //! The module declares structures to hold the address information and
 //! a set of functions to parse the raw data.
 //!
-//! Addressing system in Serbija, seems like, allows buildings without assigned number
-//! (see https://www.politika.rs/sr/clanak/398656/Srbija-bez-b-b-adresa)
+//! Addressing system in Serbija, seems like, allows buildings without assigned
+//! number (see https://www.politika.rs/sr/clanak/398656/Srbija-bez-b-b-adresa)
 //! and they are called BB (Bez Broj or without number).
 #![allow(unused)]
 
+use std::borrow::Borrow;
```

Comment:
This import is not longer needed. Please check Clippy's output.

Reply:
there's #![allow(unused)]